src/sindri/sindri.py

```diff
@@ -1,3 +1,26 @@
+"""
+## Usage
+
+```python
+# Download some sample data
+# git clone https://github.com/Sindri-Labs/sindri-resources.git
+
+# pip install sindri
+from sindri.sindri import Sindri
```

Comment:
Should this be `from sindri import Sindri`? It seems this simpler form would be nicer to promote... YMMV.

Reply:
nice catch! fixed here [3c17bec](https://github.com/Sindri-Labs/sindri-python/pull/39/commits/3c17bec522e521df13e743a968483e2369cb4547)